### PR TITLE
fix flaky joiner test

### PR DESCRIPTION
**Pull request by sumengwang**

### CODE REVIEW

File: app/hydrator/services/create/stores/config-store.js [hunk 1]
```diff
@@ -176,7 +176,12 @@ class HydratorPlusPlusConfigStore {
       const sanitize =  window.CaskCommon.CDAPHelpers.santizeStringForHTMLID;
       if (node.outputSchemaProperty) {
         try {
-          let outputSchema = JSON.parse(node.outputSchema);
+          let outputSchema 
+          if (angular.isArray(node.outputSchema)) {
+            outputSchema = JSON.parse(node.outputSchema[0].schema);
```
Comment (sumengwang) [inline app/hydrator/services/create/stores/config-store.js]: i don't know why this is happening. The joiner plugin returns an array of objects rather than a string

File: app/cdap/components/hydrator/components/TopPanel/NameAndDescription/styles.ts [hunk 2]
```diff
@@ -104,7 +104,7 @@ export const HydratorMetadata = styled.div`
     height: 185px;
     cursor: auto;
     padding: 10px;
-    width: 100%;
+    width: 92%;
```
Comment (de-lan) [inline app/cdap/components/hydrator/components/TopPanel/NameAndDescription/styles.ts]: why this change?

Comment (sumengwang) [reply app/cdap/components/hydrator/components/TopPanel/NameAndDescription/styles.ts]: it kinda solves the problem when you expand nameAndDescrption, it went out of the border

File: cypress/integration/pipeline.joiner.spec.ts [hunk 3]
```diff
@@ -14,55 +14,73 @@
...
+  nodeName: "BigQueryTable",
+  nodeType: "batchsink",
+};
+const sinkNodeId: INodeIdentifier = { ...sinkNode, nodeId: "3" };
 
 // Preview messaging
-const runPreviewMsg = 'Run preview to generate preview data';
-const noOutputDataMsg = 'Input records have not been generated';
-const successMsg = 'The preview of the pipeline has completed successfully.';
+const runPreviewMsg = "Run preview to generate preview data";
```
Comment (de-lan) [inline cypress/integration/pipeline.joiner.spec.ts]: Any specific reason we're using double quote?

File: app/hydrator/services/create/stores/config-store.js [hunk 4]
```diff
@@ -176,7 +176,12 @@ class HydratorPlusPlusConfigStore {
       const sanitize =  window.CaskCommon.CDAPHelpers.santizeStringForHTMLID;
       if (node.outputSchemaProperty) {
         try {
-          let outputSchema = JSON.parse(node.outputSchema);
+          let outputSchema
```
Comment (njbriggs) [inline app/hydrator/services/create/stores/config-store.js]: A little cleaner to make this
```
const outputSchema = angular.isArray(node.outputSchema) ?
  outputSchema = JSON.parse(node.outputSchema[0].schema) :
  outputSchema = JSON.parse(node.outputSchema);
```

File: cypress/integration/pipeline.joiner.spec.ts (hunk 3 above)
Comment (njbriggs) [reply cypress/integration/pipeline.joiner.spec.ts]: In general we prefer single quotes for strings - that and the other formatting changes make it hard to see the actual changes in the file.

**Review by de-lan — APPROVED** [review]

File: app/hydrator/services/create/stores/config-store.js [hunk 5]
```diff
@@ -176,7 +176,9 @@ class HydratorPlusPlusConfigStore {
       const sanitize =  window.CaskCommon.CDAPHelpers.santizeStringForHTMLID;
       if (node.outputSchemaProperty) {
         try {
-          let outputSchema = JSON.parse(node.outputSchema);
+          const outputSchema = angular.isArray(node.outputSchema) ?
```
Comment (seanfeld) [inline app/hydrator/services/create/stores/config-store.js:179]: don't use angular.isArray

**Review by seanfeld — APPROVED** [review]
nit but please change it. - don't use angular.isArray - can use Array.isArray or whatever else instead
